wgpu-core/src/command/mod.rs

```diff
@@ -220,6 +220,11 @@ pub struct RawRenderPass {
     targets: RawRenderTargets,
 }
 
+/// # Safety
+///
+/// This function is unsafe as there is no guarantee that the given pointer
+/// (`RenderPassDescriptor::color_attachments`) is valid for
```

Comment:
I **wish** we had a Rust type that works as a slice for Rust yet looks like a poiner + size on the C side :/

Reply:
Yeah, that would be handy. I mean we could transmute the slice to pointer + length, if it has a stable repr, but that's practically the same thing. Even if there was a way to express this, it would probably only work when calling from Rust to C. The other way around the caller could still provide bad data :/